### PR TITLE
Add surge etc to printing template variables

### PR DESCRIPTION
**Pull request by atdotde**
Signed-off-by: Robert C. Helling <helling@atdotde.de>

### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

File: core/subsurface-qt/diveobjecthelper.h [hunk 1]
```diff
@@ -15,6 +15,10 @@ class DiveObjectHelper {
 	Q_PROPERTY(int id MEMBER id CONSTANT)
 	Q_PROPERTY(int rating MEMBER rating CONSTANT)
 	Q_PROPERTY(int visibility MEMBER visibility CONSTANT)
+	Q_PROPERTY(int wavesize MEMBER wavesize CONSTANT)
```
Comment (bstoeger) [inline core/subsurface-qt/diveobjecthelper.h:18]: Now that grantlee is gone do we actually need that DiveObjectHelper thing?

**Comment by bstoeger** [comment]
Looks correct and compiles - let's merge.

**Comment by bstoeger** [comment]
Looking at the code, I'm not 100% sure, but the loop thing appears fishy to me:

Suppose that you have a list without elements (for example, we now support dives with no cylinders). Since the loop is not entered a single time, the "pos" variable will not be increased. This should output one empty loop content and then exit early when it reaches the end for, isn't it?

I'll try to fix this.

File: core/subsurface-qt/diveobjecthelper.h (hunk 1 above)
Comment (atdotde) [reply core/subsurface-qt/diveobjecthelper.h:18]: True, we don't a lot of the bells and whistles and if I built it from scratch I wouldn't go this way. But we still need to access all the stuff in the dive list via string names (as they come from the template), so I don't think we can get much leaner here.

**Comment by dirkhh** [comment]
I love the order of operation. Merged, then question.
I do that all the time :-)

In case you are wondering "what's Dirk doing these days..." --- still driving himself to tears with the Kirigami / Qt update.
@tcanabrava is now trying to help me.

**Comment by atdotde** [comment]
Yay! Welcome back @tcanabrava !

Qt update? Are you an early adopter of Qt 6.0?

**Pull request merged**